setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="cumulus",
+    version="0.1.0",
+    description="Girder API endpoints for interacting with cloud providers.",
```

Comment:
This should probably be "A RESTful API for the creation and management of HPC clusters"